.github/workflows/githib-actions.yml

```diff
@@ -41,4 +41,29 @@ jobs:
                 uses: actions/upload-artifact@v2
                 with:
                     name: Jacoco report
-                    path: build/jacocoHtml
\ No newline at end of file
+                    path: build/jacocoHtml
+    package:
+        name: Generate APK
+        runs-on: ubuntu-latest
+        steps:
+            -   name: Checkout the code
+                uses: actions/checkout@v2
+
+            -   name: set up JDK 11
+                uses: actions/setup-java@v2
+                with:
+                    distribution: 'zulu'
```

Comment:
Why zulu?)

Reply:
[Repo of `setup java` actions](https://github.com/actions/setup-java) with a variety of JDK distributors.
[Zulu jdk](https://www.azul.com/downloads/?package=jdk)

This distribution had good reviews, was often used by other people for Java 11, and most importantly, it just worked for us :))